.travis.yml

```diff
@@ -54,4 +54,4 @@ before_install: |
   sudo apt-get install -y binutils libproj-dev gdal-bin
   pip install -U pip setuptools wheel
 install: |
-  pip install -r ./dev-requirements.txt
+  pip install -e ./django_stubs_ext -r ./dev-requirements.txt
```

Comment:
Maybe we should add it to `./dev-requirements`? Things like this are hard to remember during the initial project setup. And this might complicate it for new devs!

Reply:
for sure. i had it in the wrong place in dev-requirements which was causing issues, so i tried to fix it this way